unitex/src/fr/umlv/unitex/io/GraphIO.java

```diff
@@ -97,11 +99,18 @@ public static GraphIO loadGraph(File grfFile, boolean isSentenceGraph,
 		res.grf = grfFile;
 		InputStreamReader reader;
 		if (!grfFile.exists()) {
-			if (emitErrorMessage)
-				JOptionPane.showMessageDialog(null,
-						"Cannot find " + grfFile.getAbsolutePath(), "Error",
-						JOptionPane.ERROR_MESSAGE);
-			return null;
+			if (emitErrorMessage) {
+				String[] options = { "Yes", "No" };
+				int result = JOptionPane.showOptionDialog(null,
+						"Cannot find " + grfFile.getAbsolutePath() + ". \n Create a new graph with same name?",
```

Comment:
I would like to suggest: "Graph not found! Do you want to create a new one on /foo/bar.grf?"